monitor/inclusion_checker.go

```diff
@@ -0,0 +1,156 @@
+package monitor
+
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/certificate-transparency-go"
+	"github.com/google/certificate-transparency-go/tls"
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/ct-woodpecker/storage"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var oldestUnseen = promauto.NewGauge(prometheus.GaugeOpts{
+	Name: "oldest_unincorporated_cert",
+	Help: "Number of seconds since the oldest unincorporated certificate was submitted",
```

Comment:
This isn't completely correct, the SCT isn't included in the log, it's the receipt for submission of a certificate/promise it'll be included.

Reply:
Wasn't the deal with Nimbus that they gave out multiple, different SCTs for the same certificate without incorporating that certificate multiple times?

Reply:
Yup. To boil it down further there are three elements, certificates, SCTs, and entries. A SCT is a promise that a submitted certificate will be included in the log as an entry. Multiple SCTs can exist for a single cert, which means multiple entries must be created in the log for that certificate.

Something along the lines of "since the oldest SCT that we haven't linked to a log entry was received" might work.